### ISSUE TITLE
Missing programs from Texlive

### ISSUE THREAD

**Issue by SDRausty** [termux/termux-packages#966]
There are some important programs like [`xindy`](https://github.com/jschrod/xindy.ctan), [`biber`](https://github.com/plk/biber) and `makeindex` missing from the Termux Texlive package. Can they be included?

**Comment by Grimler91** [comment]
Hi @sdrausty,
I can activate makeindex the next time I push a texlive update (which I don't want to do too often since each revision currently forces users to download every CTAN package in the installation scheme again). 
Biber (probably) requires some porting (I haven't tried building it, yet).
xindy (probably) requires quite a lot of work to get it running on termux ([see the notes here](https://www.tug.org/texlive/build.html#xindy))

Feel free to contribute and attempt to build the packages yourself! :)

I'll post here if I make any progress.

**Comment by sdrausty** [comment]
Thank you for preparing TexLive for​ Termux! If you can activate makeindex, the HoTTbook  https://github.com/sdrausty/HoTTbook might build from the Makefile onDevice as well as from it does using pdflatex with bibtex. The ~472 page book builds well onDevice, but alas without makeindex.

Building https://github.com/sdrausty/hpmor ~1985 pages, and then following along listening to `espeak`'s tember and tone digital voice; Have you tried it on?

**Comment by Grimler91** [comment]
I'll forward the praise to @rnauber since he has done ~95 % on the work of Termux's texlive package. 

I haven't encountered about HPMOR before, it seems interesting. I will be spending in total 10 hours on a train this weekend, HPMOR + espeak might come in handy 👍

**Comment by Neo-Oli** [comment]
@Grimler91 @sdrausty As a total side note. HPMOR has an excellent Fan-made Audiobook http://www.hpmorpodcast.com/?page_id=56

**Comment by sdrausty** [comment]
I am impressed.

cdd
p /sdcard/Download/
wget
http://www.hpmorpodcast.com/wp-content/uploads/episodes/HPMoR_Chap_1.mp3

Thank you for the link to the audio book!

**Comment by Grimler91** [comment]
@sdrausty, makeindex is now available with the latest texlive update (#1072). I haven't tested it though so please report bugs if encountered.

**Comment by sdrausty** [comment]
@Grimler91 Thank you for developing this package! 

https://github.com/sdrausty/hpmor builds into a wonderful 1985 page PDF with one command `make`.

![screenshot_20170611-165107](https://user-images.githubusercontent.com/27742457/27014406-31d64500-4ec6-11e7-9272-7376082d15f7.png)

**Comment by sdrausty** [comment]
@Grimler91 `makeindex`works well on https://github.com/Tufte-LaTeX/tufte-latex Thank you 😀

**Comment by Grimler91** [comment]
@sdrausty That's great to hear :)

**Comment by qoheniac** [comment]
Has anyone tried to build biber? What is the problem with it?

**Comment by Grimler91** [comment]
@qoheniac I haven't tried it since my last message here, but the problem bqck then

**Comment by qoheniac** [comment]
Did you read the instructions in section 4.4 of the documentation [here](https://sourceforge.net/projects/biblatex-biber/files/biblatex-biber/current/documentation/biber.pdf)?

**Comment by Grimler91** [comment]
Yes. Problem is that termux [isn't a full gnu/linux system](https://wiki.termux.com/wiki/Differences_from_Linux) and perls package manager doesn't handle this as nicely as for example pip.

But please try building it, the situation might have improved during this year.

**Comment by qoheniac** [comment]
I just tried installing the required perl packages via cpanm and there were three making trouble:

1. ExtUtils:LibBuilder
2. LWP:Protocol:https
3. Text::BibTeX

It seems like in all cases there are problems with linking C libraries provided by the packages themselves but I am no expert on that.

**Comment by qoheniac** [comment]
The problem with LWP::Protocol::https was that I was missing the openssl-tool package. Text::BibTeX seems to be more complicated: I tried to build btparse from [source](http://www.gerg.ca/software/btOOL/) but all tests failed.
